README.md

```diff
@@ -248,6 +252,11 @@ Here is the outline of the configuration options, descriptions of each are below
         importScripts,
         cacheId
     },
+    copy: {
+        js,
```

Comment:
If these properties aren't actually there by default should we list them here? Another option could be to add them as empty properties on the config object?